api-reference/beta/resources/orgcontact.md

```diff
@@ -47,11 +48,12 @@ Here is a JSON representation of the resource
 |city|String||
```

Comment:
I know it's not the target of this PR, but it would be nice to fill in the descriptions.  See https://msdn.microsoft.com/en-us/library/azure/ad/graph/api/entity-and-complex-type-reference#contact-entity for existing descriptions in AAD Graph.